src/tests/Directory.Build.props

```diff
@@ -74,6 +74,7 @@
     <RuntimeFlavor Condition="'$(RuntimeFlavor)' == ''">coreclr</RuntimeFlavor>
 
     <RestoreDefaultOptimizationDataPackage Condition="'$(RestoreDefaultOptimizationDataPackage)' == ''">false</RestoreDefaultOptimizationDataPackage>
+    <PortableBuild Condition="'$(PortableBuild)' == '' and '$(DotNetBuildFromSource)' == 'true'">false</PortableBuild>
```

Comment:
Same as https://github.com/dotnet/runtime/pull/81480/files#r1093359017.

I am not sure why we need this here, when it's already in the top-level `Directory.Build.props`.